lib/diagnostic.rb

```diff
@@ -100,13 +105,14 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.slice!(2, 2)
```

Comment:
This would work, but remember about `pop`

`arr.pop(2)` Would remove the last 2 items in the array